taxi_service/settings.py

```diff
@@ -20,9 +20,8 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-8ovil3xu6=eaoqd#-#&ricv159p0pypoh5_lgm*)-dfcjqe=yc"
-)
+SECRET_KEY = "django-insecure-8ovil3xu6" \
```

Comment:
why did you change it? () is better then \

Reply:
I didn't change anything, maybe I suddenly pressed something

Reply:
wait, I changed it because I applied black to the entire project, after which it didn't pass through flake8